examples/with-lokijs/store/index.js

```diff
@@ -0,0 +1,93 @@
+var db = null
+export const state = () => ({
+  id: "",
+  users: []
+})
+
+export const getters = {
+  getIdMode(state) {
+    return {
+      id: state.id,
+      isCpMode: state.isCpMode
+    }
+  }
+}
+
+export const mutations = {
+  setUSers(state, users) {
+    state.users = users
+  }
+}
+
+export const actions = {
+  async nuxtServerInit({ dispatch }) {
+    console.log("STARTED SERVER")
+
+    await dispatch("INIT_DB")
+  },
+  async INIT_DB({ dispatch }) {
+    if (process.server) {
+      const res = await new Promise(resolve => {
+        const Loki = require("lokijs")
+        db = new Loki("quickstart.db", {
```

Comment:
Doing this inside nuxtServerInit or plugin function makes Loki to be loaded per each SSR request. It has to be done in a global scope.

Reply:
Plugin can inject `$loki` into context so other store action would benefit from it.